src/main/java/no/nav/fo/veilarbregistrering/oppgave/adapter/OppgaveDto.java

```diff
@@ -20,4 +11,82 @@
     private String prioritet;
     private String tildeltEnhetsnr;
 
+    public OppgaveDto(String aktoerId, String beskrivelse, String tema, String oppgavetype, String fristFerdigstillelse, String aktivDato, String prioritet, String tildeltEnhetsnr) {
```

Comment:
Hvis vi bruker constructor i "kjørekoden" vår, så tror jeg vi kan droppe set-metodene her. Vi bruker denne DTOen kun på vei ut mot Oppgave, og da tror jeg ikke noe rammeverk trenger å sette noen verdier.

Er litt mer usikker på get-erne, men det kunne vi jo ha testet - om rammeverket uansett benytter reflection for å snakke med feltene direkte.

Reply:
Godt observert! Jeg skal ta en liten titt på dette etter jeg har fått ut denne PRen.